lib/src/timing_specifiers.dart

```diff
@@ -0,0 +1,16 @@
+class StartupTimingSpecifier {
```

Comment:
Not my favorite name.

Reply:
I was thinking "mark" or "indicate" for the verb for use in the method name, and then this could be "type" or un-suffixed or something.

```dart
markStartupTiming(StartupTiming.firstEditable);
indicateStartupTiming(StartupTimingType.firstEditable);
```